.travis.yml

```diff
@@ -4,17 +4,34 @@ go_import_path: github.com/KohlsTechnology/eunomia
 go:
 - "1.12"
 env:
+  global:
   - GO111MODULE=on
+  - CHANGE_MINIKUBE_NONE_USER=true
+  matrix:
+  - OPERATOR_SDK_VERSION=v0.9.0 KUBE_VERSION=v1.14.2
 
 before_install:
   - travis_retry go mod vendor
   - go get golang.org/x/lint/golint
 
+before_script:
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl
+- sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+- sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
+- curl -LO https://git.io/get_helm.sh && chmod 700 get_helm.sh && ./get_helm.sh
+- curl -OJL https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu && chmod +x operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk && rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
```

Comment:
Since this line is quite long, can we break up the `&&`'s into just separate lines?

Reply:
@etsauer fixed it!